src/handlers/issue-closed.ts

```diff
@@ -0,0 +1,16 @@
+import { Result } from "../proxy";
+import { Context } from "../types/context";
+import { EnvConfigType } from "../types/env-type";
+
+/**
+ * On issue closed, we want to delete the entry to stop watching this issue.
+ */
+export async function handleIssueClosed(context: Context, env: EnvConfigType): Promise<Result> {
+  const {
+    adapters: {
+      supabase: { repositories },
+    },
+  } = context;
+  await repositories.delete(context.payload.issue.html_url);
```

Comment:
You just reminded me that GitHub has some type of feature to associate metadata with each repository. Perhaps we can write whatever data we need here for minimizing runs?

> ## Custom Properties for Repositories
> GitHub also allows organizations to manage custom properties for their repositories. These properties can be used to add metadata such as compliance frameworks, data sensitivity, or project details. Custom properties can be viewed by people with read permissions to the repository and can be used to search and filter repositories within an organization.
> ### Adding Custom Properties
> Navigate to the repository's settings.
> In the "Code and automation" section, click "Custom properties".
> Click "New property" to add a new custom property.
> Fill in the name, description, and type of the property.
> Optionally, set permissions and default values.
> Save the property.
> ### Setting Values for Custom Properties
> Navigate to the organization's settings.
> In the "Code, planning, and automation" section, click "Repository" and then "Custom properties".
> Select the repositories and click "Edit properties".
> Set the values for each property and save the changes.
> These methods provide a comprehensive way to access and manage GitHub repository metadata, whether through automated workflows, API calls, or custom properties within an organization.